test/zipWith.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+
+
+test('zipWith', function() {
+
+  eq(typeof S.zipWith, 'function');
+  eq(S.zipWith.length, 3);
+  eq(S.zipWith.toString(), 'zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c');
+
+  eq(S.zipWith(S.add, [1, 2, 3, 4], [3, 2, 1]), [4, 4, 4]);
+  eq(S.zipWith(S.add, [1, 2, 3], [3, 2, 1, 0]), [4, 4, 4]);
+
+  function concat(a) {
+    return function(b) {
+      return a + b;
+    };
+  }
+  eq(S.zipWith(concat, ['a', 'b'], ['x', 'y', 'z']), ['ax', 'by']);
+
+  function pair(a) {
+    return function(b) {
+      return [a, b];
+    };
+  }
```

Comment:
Let's move this definition to __test/internal/pair.js__.

Reply:
Neat!!